typescript/cli/src/send/message.ts

```diff
@@ -146,6 +146,9 @@ async function executeDelivery({
     logBlue(`Sent message from ${origin} to ${recipient} on ${destination}.`);
     logBlue(`Message ID: ${message.id}`);
     log(`Message: ${JSON.stringify(message)}`);
+
+    await core.relayMessage(message);
```

Comment:
Should we check if the destination is a core chain and let regular relay handle those?

Reply:
imo self-relay is analogous to `relay --force` so it makes sense to bypass regular relaying

Reply:
added an explicit --selfrelay flag